### PR TITLE
AIEye Fixes - Fixes #7197 and Fixes #7559

### LINKED ISSUE #7197: Autocapitalization is always on when inside the AI Eye
### Labels

[BUG]

### Description

You can't turn off autocapitalization in your messages when you are inside of an AI eye

### Steps to reproduce

1. Spawn as AI
2. Say something with autocapitalization off, it will behave correctly
3. Move around to exit your core into the AI eye
4. Say something
5. See bug

### Expected Behavior

Messages to not be autocapitalized when inside the AI eye.

### Additional Information & Screenshots

image

mildly annoying

### LINKED ISSUE #7559: AI Eye can stack ores
### Labels

[BUG]

### Description

You can stack ores on the floor as an AI eye. That ain't right.

### Steps to reproduce

1. Be AIEye
2. Have there be some scattered ores on the floor (ie, eject from Rockbox)
3. click and drag an or onto a tile to stack

### Expected Behavior

Invisible ghost eyeballs can't stack rocks, so ideally nothing should happen

### Additional Information & Screenshots

_No response_

### PR DESCRIPTION
**Pull request by amylizzle**
[BUG]
## About the PR 

Fixes #7197 and #7559
Blob overminds and AI eyes can't stack ores anymore, and receive a message similar to ghosts when they try.
Fixed a minor code error which meant auto-capitalisation was always on for the AI eye
Adds a logging message for admins when something tries to gib() the AI eye.

## Why's this needed? 

Bugs are bad

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
One day we'll fix all the AIEye jank. Probably don't need a changelog for these though

File: code/mob/living/intangible/ai-camera.dm [hunk 1]
```diff
@@ -282,6 +282,13 @@
 	resist()
 		return 0 //can't actually resist anything because there's nothing to resist, but maybe the hot key could be used for something?
 
+	//death stuff that should be passed to mainframe
+	gib(give_medal, include_ejectables) //this should be admin only, I would hope
+		message_admins("something gibbed the AI - if this wasn't an admin action, something has gone badly wrong")
+		return mainframe.gib(give_medal, include_ejectables)
```
Comment (TobleroneSwordfish) [inline code/mob/living/intangible/ai-camera.dm:288]: I'm not sure this is a good idea, there's a lot of M.gib() calls in the code, and the possibility that one of them is going to hit the eye and gib an AI for flying over something seems bad.

Comment (amylizzle) [reply code/mob/living/intangible/ai-camera.dm:288]: There's a reason for this, and it's that I called gib on the player instance with the player panel and it caused the AI eye to spew gibs everywhere and get into a weird state.

Maybe we should just log it and return instead?

**Comment by MetricDuck** [comment]
If you want to link multiple issues you have to put "Fixes" in front of each, ie : "Fixes #7197 and Fixes #7559"

**Review by pali6 — APPROVED** [review]

**Pull request merged**